Sources/Networking/HTTPClient/HTTPClient.swift

```diff
@@ -532,10 +542,13 @@ extension HTTPRequest {
 
         if #available(iOS 13.0, macOS 10.15, tvOS 13.0, watchOS 6.2, *),
            verificationMode.isEnabled,
-           self.path.supportsSignatureVerification,
-           let body = self.requestBody {
+           self.path.supportsSignatureVerification {
+            result += HTTPClient.headerParametersForSignatureHeader(with: defaultHeaders)
```

Comment:
FFTI I was wondering whether we should pass the `result` headers instead of only the `defaultHeaders` to future-proof it in case we need to sign any of the other headers... This should work for now though.

Reply:
Yeah if we need to that will get caught by tests, so we can change this to support that.